packages/@sanity/form-builder/src/inputs/ArrayInput/ItemValue.js

```diff
@@ -210,6 +211,8 @@ export default class RenderItemValue extends React.Component<Props> {
       )
     }
 
+    const dialogSize = 'medium'
```

Comment:
This is supposed to be hard coded?

Reply:
Yes. Until we have a resolver for size. This sets the size for both popover and defaultdialog